cmd/parca-agent/main.go

```diff
@@ -296,24 +306,170 @@ func run(logger log.Logger, reg *prometheus.Registry, flags flags) error {
 		}
 		if r.URL.Path == "/" {
 			w.Header().Set("Content-Type", "text/html; charset=utf-8")
-			statusPage := template.StatusPage{}
+			statusPage := template.StatusPage{
+				ProfilingInterval:   flags.ProfilingDuration,
+				ProfileLinksEnabled: flags.LocalStoreDirectory == "",
+			}
+
+			processReports := map[string]map[int]error{}
 
 			for _, profiler := range profilers {
 				statusPage.ActiveProfilers = append(statusPage.ActiveProfilers, template.ActiveProfiler{
 					Name:           profiler.Name(),
-					Interval:       flags.ProfilingDuration,
 					NextStartedAgo: time.Since(profiler.LastProfileStartedAt()),
 					Error:          profiler.LastError(),
 				})
+
+				processReports[profiler.Name()] = profiler.ProcessReports()
 			}
-			err := template.StatusPageTemplate.Execute(w, statusPage)
+
+			procDirEntries, err := os.ReadDir("/proc")
```

Comment:
Even though it does the same thing under the hood, I would love to unify things and use them https://pkg.go.dev/github.com/keybase/go-ps?utm_source=gopls#Processes to discover processes.